Pages/ProfilePage.cs

```diff
@@ -0,0 +1,130 @@
+using System;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using SeleniumExtras.WaitHelpers;
+
+namespace QA_Mars_OnboardingTaskSpecflow.Pages
+{
+    public class ProfilePage
+    {
+        private readonly IWebDriver driver;
+        private WebDriverWait wait;
+        private readonly By btnAddNewLangauageLocator = By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/thead/tr/th[3]/div");
+        private readonly By txaLanguageLocator = By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/div/div[1]/input");
```

Comment:
please change all variables to return IWebElement rather than BY for example  By txaLanguageLocator = By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/thead/tr/th[3]/div")

to 

private IWebElement txaLanguageLocator = driver.FindElement( By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/thead/tr/th[3]/div")));

Reply:
you can see 
https://docs.specflow.org/projects/specflow/en/latest/ui-automation/Selenium-with-Page-Object-Pattern.html 
 for example

Reply:
please fix all pages